recipes/simplecosine/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Where did this license file come from? I don't see it upstream?

Reply:
@dopplershift I actually used the same license file as [dedupe ](https://github.com/dedupeio/dedupe) since simplecosine didn't have one, but they were created by the same people/for the same solution. Using grayskull to create the recipe, it prompted to input a license file. If there is a better way of doing this, let me know.

Reply:
@dopplershift My main goal is actually to get [dedupe ](https://pypi.org/project/dedupe/) as a library in conda-forge, but have noticed it has some dependencies that conda-forge doesn't have. I decided to upload each dependency one by one because some are dependent on eachother. Is this the best process? Is there a better one?

Reply:
One-by-one or one big PR are both fine. The bigger problem is that strictly speaking, without a license in the upstream repository, we don't actually have the rights to redistribute this package. By default in US copyright law, the owners of the intellectual property maintain all rights. So in order to add this we need upstream to have an explicit license.

Reply:
@dopplershift Thanks for your help! I'll reach out to them to see if they would create a license for this.

Reply:
@dopplershift The license link provided in simplecosine pypi documentation is https://opensource.org/licenses/mit-license.php, which shows the same license that I have provided here. The difference is the line "Copyright (c) 2014 Forest Gregg, Derek Eder, DataMade and Contributors". Would it be fine if I remove that line? If not, can I provide the link that was provided in the simplecosine pypi documentation?

Reply:
Yeah, remove that line, then it will match what upstream has attempted to do.